app/components/lg-dob-selector.js

```diff
@@ -0,0 +1,41 @@
+/* global moment */
+
+import Ember from 'ember';
+import {dobOptions} from 'lion-guardians/utils/units';
+
+export default Ember.Component.extend({
+  dobOptions: dobOptions,
+  selectedDobOption: dobOptions[0].value,
+  selectedDob: null,
+  isSearching: false,
+  isEditing: false,
```

Comment:
@redyaffle I think it would be better to make a new component for date searching, which either sublcasses from this one or just copies over some of the code we need. something like `lg-dob-search-selector` There are some subtle but important differences between the drop down for setting lion age and searching
 - We need an `all` option at the top of the list to clear the age search
 - We need to spit out a pair of dates instead of just one. (more on that below in my comment in lg-lion-search.js)